book/website/communication/peer-review/peer-review-conflicts.md

```diff
@@ -0,0 +1,84 @@
+(cm-pr-conflicts)=
+# Conflict of Interest
+
+This section provides introduces and provides examples of conflicts of interest in the context of research and academic publishing. 
+
+## Context
+Imagine if a researcher with a second job at a consulting business publishes a paper that shows increased positive outcomes from certain approaches in their field. 
+Later we find out that the consulting business actively promotes and sells those approaches.
+Believe it or not this is a common problem in scientific publishing, and raises the relevance of declaring the so-called conflict of interest {cite:ps}`weinkle2024coi`.
+
+## What?
+To maintain trust and integrity of scientific publishing in the global era, it is imperative to identify conflicting biases (conscious or unconscious) across all participants in the peer-reviewed publication process - including peer reviewers, editors, and the editorial board members of the journals. 
+A “conflict of interest” or “competing interest” (COI) refers to situations that are, or could be perceived to interfere with the objective presentation, review, or publication of a piece of work {cite:ps}`DeGruyter2024coi`.
```

Comment:
```suggestion
A “competing interest” or “conflict of interest” (COI) refers to situations that are, or could be perceived to interfere with the objective presentation, review, or publication of a piece of work {cite:ps}`DeGruyter2024coi`.
```

Perhaps also a sentence on how objectivity is a thing that is difficult to achieve anyway since research is carried out by humans?